docs/guides/dynamic-circuits-considerations.ipynb

```diff
@@ -32,27 +32,33 @@
     "<span id=\"buffer-overflow\"></span>\n",
-    "- **Buffer overflow error**. One of the most common limitations users can encounter when executing dynamic circuits is a buffer overflow during measurement result collection. The measurement buffer on the control system can only store a limited number of measurement results. If the circuit performs many measurements in rapid succession, the buffer might fill before the control system has time to stream the results to the host, causing a job failure.\n",
+    "- **Buffer overflow error**: One of the most common limitations users can encounter when executing dynamic circuits is a buffer overflow during measurement result collection. The measurement buffer on the control system can only store a limited number of measurement results. If the circuit performs many measurements in rapid succession, the buffer might fill before the control system has time to stream the results to the host, causing a job failure.\n",
     "\n",
     "  If you encounter the error `\"RuntimeJobFailureError: unable to retrieve job result. A buffer overflow occurred during result collection...\"`, try these strategies to mitigate this issue:\n",
-    "  - **Increase `rep_delay` (delay between shots):** This adds time between shots, giving the control system more time to offload measurement results to the host and clear the buffer. This is the most direct fix and does not impact shot quality, though it will increase total job execution time.\n",
-    "  - **Insert `qc.delay()` after measurements:** Add a delay instruction between measurements within a circuit to give the system time to stream results and prevent the buffer from filling.\n",
-    "  **Note:** this approach increases the circuit duration, which might reduce shot quality due to qubit decoherence.\n",
+    "  - **Increase `rep_delay` (delay between shots)**: This adds time between shots, giving the control system more time to offload measurement results to the host and clear the buffer. This is the most direct fix and does not impact shot quality, though it will increase total job execution time.\n",
+    "  - **Insert `qc.delay()` after measurements**: Add a delay instruction between measurements within a circuit to give the system time to stream results and prevent the buffer from filling.\n",
+    "     <Admonition type=\"note\">\n",
+    "  This approach increases the circuit duration, which might reduce shot quality due to qubit decoherence.\n",
+    "     </Admonition>\n",
+    "\n",
+    "  ## Other considerations\n",
+    "\n",
+    "  [Stretch](/docs/guides/stretch) values are resolved to integer multiples of the fundamental scheduling unit for the backend. When resolving uneven or non-integer results, individual delay values that are parameterized by the same stretch variable and equation might resolve to different durations. The remainder is added to the first delay parameterized by the stretch.   \n",
```

Comment:
Why is this here and not the stretch page? Especially since the stretch page has a code example.